Makefile

```diff
@@ -15,11 +15,14 @@ LCC = $(GBDK_HOME)bin/lcc
 PROJECTNAME    = WORDLE
 
 BINS	    = $(PROJECTNAME).gb
-CSOURCES   := $(wildcard *.c)
+CSOURCES   := main.c word-db.c words.c
```

Comment:
This breaks if you compile from a fresh checkout since words.c does not exist and needs to be generated from the python script so it needs to be specified manually but you can't have *.c and `words.c` because that will include it twice so in that case the CSOURCES needs to be "all files *.c except words.c" + "words.c" explicitly.